src/components/Year.jsx

```diff
@@ -39,10 +39,18 @@ export const Year = ({ year, colorScheme }) => {
                           propName="title"
                           validate={_.isString}
                         />
-                        <Button
-                          color="link"
-                          onClick={() => year.addTerm()}
-                        >+</Button>
+                        <Dropdown
+                          isOpen={year.dropdownOpen}
+                          toggle={year.toggleDropdown}
+                        >
+                          <DropdownToggle>
+                            <i className="fa fa-ellipsis-h" aria-hidden="true"></i>
+                          </DropdownToggle>
+                          <DropdownMenu>
+                            <DropdownItem onClick={() => year.addTerm()}>Add Term</DropdownItem>
+                            <DropdownItem onClick={() => prompt('todo. use trash can for now, pleb')}>Delete Year</DropdownItem>
```

Comment:
I'll implement this after we put in the edit course modal, that way i can use that code as an example

Reply:
You can also look at the handler on the Trash component, might be a bit more useful